setup.py

```diff
@@ -95,9 +95,10 @@
     "sentry-sdk>=0.10.2",
     # Avoid version 4.3 it breaks the emails with datetime
     # https://github.com/celery/celery/pull/5606
-    "celery>=4.2.1<4.3.0",
+    "celery>=4.2.1,<4.3.0",
```

Comment:
You can remove this now, you might have a conflict when rebasing...

Can you please please keep invenio deps in the `# extra` "section", just after `invenio-pidrelations`?
Thanks!